azure-pipelines.yml

```diff
@@ -0,0 +1,52 @@
+trigger:
+  tags:
+    include:
+    - '*'
+  branches:
+    include:
+    - master
+pr:
+  branches:
+    include:
+    - master
+
+jobs:
+
+- job:
+  strategy:
+    matrix:
+      Mojave-Xcode-10.3:
+        IMAGE_POOL: 'macOS-10.14'
+        XCODE_VERSION: '10.3'
+      Mojave-Xcode-11.0:
+        IMAGE_POOL: 'macOS-10.14'
+        XCODE_VERSION: '11'
+  pool:
+    vmImage: $(IMAGE_POOL)
+
+  steps:
+  - script: |
+      set -e
+      sudo xcode-select --switch "/Applications/Xcode_$(XCODE_VERSION).app/Contents/Developer"
+    displayName: "$(IMAGE_POOL) - Xcode $(XCODE_VERSION)"
+    
+  - script: |
+      set -e
+      ruby -v
+      bundler -v
+    displayName: "ruby info"
+
+  - script: |
+      set -e
+      echo -e "install: --no-document --env-shebang\nupdate:  --no-document --env-shebang" > ~/.gemrc
+      bundle install
+    displayName: "bundle install"
+    
+  - script: cd ./CalSmokeApp && bundle install
+    displayName: "CalSmokeApp bundle install"
+
+  - script: cd ./CalSmokeApp && make app-cal
```

Comment:
instead of `cd` use `workingDirectory` property

Reply:
- script: 'bundle install'
  workingDirectory: 'CalSmokeApp'
  displayName: CalSmokeApp bundle install